src/backend/router/index.ts

```diff
@@ -41,25 +55,27 @@ export const appRouter = trpc
           source: input.source,
           xp: input.xp,
           deckId: input.deckId,
-        }
+        },
       });
       return newCard;
-    }
+    },
   })
   .mutation("add-deck", {
     input: z.object({
       name: z.string(),
-      type: z.string()
+      type: z.string(),
+      userId: z.string(),
```

Comment:
Add user id to adding new decks.

Reply:
Little bit of a conflict with my change, but that may or may not actually go in and/or will be easy to fix.